R/capacity/health-inequalities/england/workforce/adult-social-care-vacanies.R

```diff
@@ -0,0 +1,170 @@
+# ---- Load libraries ----
+library(tidyverse)
+library(readxl)
+library(httr)
+library(geographr)
+library(sf)
+
+# ---- Load data ----
+source("R/utils.R")
+
+lad_lookup <-
+  boundaries_lad |>
+  st_drop_geometry()
+
+lookup_counties_lad <-
+  lookup_counties_ua_lad |>
+  select(-lad_name, -county_ua_code)
+
+# 2019 LAD populations
+pop_lad <- geographr::population_lad |>
+  select(
+    lad_code,
+    total_pop = total_population
+  )
+
+GET(
+  "https://www.skillsforcare.org.uk/adult-social-care-workforce-data/Workforce-intelligence/documents/Raw-data/LA-data-download-Oct-2021.xlsx",
```

Comment:
This link is also unfortunately no longer working

Reply:
Fixed in https://github.com/britishredcrosssociety/resilience-index/pull/6/commits/2aae0a850b578b217a27a6b1c5312187027b1f07